core/mpc/mpcKeysign.ts

```
@@ -0,0 +1,79 @@
+          publicKey,
+          '', // chainPath is only used for ECDSA right now
+          this.keysignCommittee,
+          this.isInitiateDevice,
+          keyshare
+        )
+        EventsEmit('EDDSA')
+        return await schnorrKeysign.startKeysign(messagesToSign)
+      },
+    })
+    return result
+    console.log('startKeysign done')
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)